docs/extending/reusable_back_office_templates.md

```diff
@@ -4,12 +4,38 @@ When you extend the Back Office, you can use base Twig templates for commonly us
 
 The available templates are:
 
+- `@ibexadesign/ui/component/table/alert.html.twig`
 - `@ibexadesign/ui/component/table/table.html.twig`
 - `@ibexadesign/ui/component/tab/tabs.html.twig`
 
 To use the components, [`embed`](https://twig.symfony.com/doc/3.x/tags/embed.html) them in templates.
 With `embed` you can override blocks that are defined inside the included template.
 
+## Alerts
+
+The alert component has the following properties:
+
+- `type` - available types of alert: error, info, success and warning
+- `icon_path` - overrides the path to the default icons set, can be modified
```

Comment:
```suggestion
- `icon` - name of the icon, which will be taken from the default icon set
- `icon_path` - full icon path, in case you do not want to use an icon from the default icon set
```